index.js

```diff
@@ -11,7 +11,7 @@ const ssmPath = process.env.KONG_ADMIN_TOKEN_NAME;
 const kongAdminUrl = process.env.KONG_ADMIN_URL;
 const taskDir = process.env.LAMBDA_TASK_ROOT
 
-exports.handler = async function handler({ workspace, key }) {
+exports.handler = async function handler({ workspace, key, preserve = true }) {
```

Comment:
can you default to false so that existing behaviour is maintained?
```suggestion
exports.handler = async function handler({ workspace, key, preserve = false }) {
```

Reply:
Updated